torchgeo/trainers/classification.py

```diff
@@ -179,6 +180,19 @@ def validation_step(  # type: ignore[override]
         self.log("val_loss", loss, on_step=False, on_epoch=True)
         self.val_metrics(y_hat_hard, y)
 
+        if batch_idx < 10:
```

Comment:
Why batch_idx < 10?

Reply:
Plotting doesn't need to happen for every batch, this makes it just do the first 10